.github/workflows/basic.yml

```diff
@@ -119,6 +120,13 @@ jobs:
             compiler: {c: gcc, cxx: g++}
             shared_library: 'OFF'
             os_provider: 'OFF'
+          # test level_zero_provider='OFF'
+          - os: 'ubuntu-22.04'
+            build_type: Release
+            compiler: {c: gcc, cxx: g++}
+            shared_library: 'OFF'
+            os_provider: 'ON'
+            level_zero_provider: 'OFF'
```

Comment:
You have to add `level_zero_provider` variable to all includes, otherwise you get an empty string for this value: https://github.com/oneapi-src/unified-memory-framework/actions/runs/8202777832/job/22434336069

Reply:
I also added "pool tracking" for the same reason